src/main/resources/application.conf

```
@@ -0,0 +1,14 @@
+app{
+  server {
+    port = 8088
+    host = "localhost"
+  }
+  db {
+    provider = "postgres"
+    driver = "org.postgresql.Driver"
+    url = "jdbc:postgresql://localhost/supplier_portal"
+    user = "postgres"
+    password = "admin"
```

user & password shouldn't be exposed in production repositories. Basically they're lived empty
```suggestion
    user = ""
    password = ""
```